velox/exec/TopNRowNumber.h

```diff
@@ -150,7 +156,22 @@ class TopNRowNumber : public Operator {
   bool isNewPartition(
       const RowVectorPtr& output,
       vector_size_t index,
-      SpillMergeStream* next);
+      const SpillMergeStream* next);
+
+  // Utility method to compare values from startColumn to endColumn for
+  // 'next' row from SpillMergeStream with current row of output (at index).
+  bool compareSpillRowColumns(
+      const RowVectorPtr& output,
+      vector_size_t index,
+      const SpillMergeStream* next,
+      vector_size_t startColumn,
+      vector_size_t endColumn);
+
+  // Computes next rank value for spill output.
+  void computeNextRankInSpill(
+      const RowVectorPtr& output,
+      vector_size_t index,
+      const SpillMergeStream* next);
 
   // Sets nextRowNumber_ to rowNumber. Checks if next row in 'merge_' belongs to
```

Comment:
@aditi-pandit Can you also update this commentto use `nextRank_ `to replace the `nextRowNumber_ `?

Reply:
Done.